tasks/eth/base-003-holocene-fp-upgrade/input.json

```diff
@@ -0,0 +1,67 @@
+{
+  "chainId": 8453,
+  "metadata": {
+    "name": "Base Mainnet Holocene - Proof Contract Upgrades",
+    "description": "Upgrades the `MIPS.sol`, `FaultDisputeGame.sol`, and `PermissionedDisputeGame.sol` contracts for Holocene."
```

Comment:
I don't see where the `MIPS.sol` is upgraded in this task.

Reply:
Same response as last time, MIPS isn't upgraded directly at a proxy, but implicitly because the new games reference a new MIPS implementation.

Reply:
Thanks! Sorry, I didn't saw the previous answer that make sense to me now :)